evil-collection-neotree.el

```diff
@@ -0,0 +1,100 @@
+;;; evil-collection-neotree.el --- Evil bindings for neotree -*- lexical-binding: t -*-
+
+;; Copyright (C) 2017 Pierre Neidhardt
+
+;; Author: Maximiliano Sandoval <msandova@protonmail.com>
+;; Maintainer: James Nguyen <james@jojojames.com>
+;; Pierre Neidhardt <ambrevar@gmail.com>
+;; URL: https://github.com/jojojames/evil-collection
+;; Version: 0.0.1
+;; Package-Requires: ((emacs "25.1"))
+;; Keywords: evil, neotree, tools
+
+;; This file is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published
+;; by the Free Software Foundation; either version 3, or (at your
+;; option) any later version.
+;;
+;; This file is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+;;
+;; For a full copy of the GNU General Public License
+;; see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+;; Evil bindings for `neotree'
+
+;;; Code:
+
+(require 'evil)
+(require 'neotree)
+
+;; TODO: Compare with spacemacs keybindings.
+;; https://github.com/syl20bnr/spacemacs/blob/bd7ef98e4c35fd87538dd2a81356cc83f5fd02f3/layers/%2Bspacemacs/spacemacs-ui-visual/packages.el
+
+;; Maybe there are better uses of H, K, L,
+
+(defun evil-collection-neotree-setup ()
+  "Set up `evil' bindings for `neotree'."
+
+  (evil-set-initial-state 'neotree-mode 'normal) ;; Neotree start in normal by default.
+
+  (evil-define-key 'normal neotree-mode-map
+    
+    (kbd "<return>")	(neotree-make-executor
+			 :file-fn 'neo-open-file
+			 :dir-fn  'neo-open-dir)
+    (kbd "<tab>")	(neotree-make-executor
+			 :dir-fn  'neo-open-dir)
+    "z"		(neotree-make-executor
+		 :dir-fn  'neo-open-dir)
+    "gd"	(neotree-make-executor
+		 :dir-fn 'neo-open-dired)
+    "gD"	(neotree-make-executor
+		 :dir-fn 'neo-open-dired)
+    "go"	(neotree-make-executor
+		 :file-fn 'neo-open-file
+		 :dir-fn  'neo-open-dir)
+    "gO"	'neotree-quick-look
```

Comment:
Can you re-align these so that there's no alignment? (e.g. no variable spaces between key and function.

Reply:
Do you mean?
```lisp
"go"	        (neotree-make-executor
		 :file-fn 'neo-open-file
		 :dir-fn 'neo-open-dir)
"gO"	'neotree-quick-look
```